.gitignore

```diff
@@ -1,11 +1,166 @@
-# You can keep your repository small and manageable with a .gitignore file.
-# Prevent build artifacts, large binary files, and other large file types
-# such as log files, media files, and executables from slowing down your workflow.
-#
-# If you need to keep large files in Bitbucket, consider introducing
-# Git Large File Storage (Git LFS) as part of your workflow.
-#
-# Learn more about .gitignore:
-#     https://www.atlassian.com/git/tutorials/saving-changes/gitignore
-# Learn more about Git LFS for large files:
-#     https://www.atlassian.com/git/tutorials/git-lfs
\ No newline at end of file
+# Byte-compiled / optimized / DLL files
```

Comment:
Why are you adding this? This has nothing to do with the PR?

Reply:
fixed